src/org/ods/services/GitService.groovy

```diff
@@ -74,13 +74,16 @@ class GitService {
         } else if (branch.startsWith('release/')) {
             def list = branch.drop('release/'.length()).tokenize('-')
             "${list[0]}-${list[1]}"
+        } else if (branch.startsWith('release-ods-generated/')) {
```

Comment:
@michaelsauter not sure about this code portion. Seems like the block in L74 is a left-over before `ODS_GIT_TAG_BRANCH_PREFIX` was introduced? Think we'll only ever need one block that refers to a constant.

Reply:
AFAIk this is only used by components, to extract a Jira issue ID (from Atlassian-suggested branch patterns). The code path is not used from the orchestration pipeline, so we don't need any changes here.

Reply:
Reverted this change.